src/globals.css

```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.custom-gradient {
```

Comment:
refer [these](https://tailwindcss.com/docs/adding-custom-styles) docs to make better custom styles